### PR TITLE
Prepares release 2.9.0

### PR DESCRIPTION
**Pull request by MattHJensen**
@Peter-Metz, could you look over this?

### CODE REVIEW

File: docs/tc_history.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!-- *** NEVER EDIT THIS FILE BY HAND *** -->
-<!-- *** INSTEAD EDIT docs/tc_history.htmx *** -->
+<!-- *** INSTEAD EDIT /Users/matthewjensen/Projects/Tax-Calculator/docs/tc_history.htmx FILE *** -->
```
Comment (Peter-Metz) [inline docs/tc_history.html]: Might want to revert file path

File: docs/tc_license.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!-- *** NEVER EDIT THIS FILE BY HAND *** -->
-<!-- *** INSTEAD EDIT docs/tc_license.htmx *** -->
+<!-- *** INSTEAD EDIT /Users/matthewjensen/Projects/Tax-Calculator/docs/tc_license.htmx FILE *** -->
```
Comment (Peter-Metz) [inline docs/tc_license.html]: Same here @MattHJensen

File: docs/tc_overview.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!-- *** NEVER EDIT THIS FILE BY HAND *** -->
-<!-- *** INSTEAD EDIT docs/tc_overview.htmx *** -->
+<!-- *** INSTEAD EDIT /Users/matthewjensen/Projects/Tax-Calculator/docs/tc_overview.htmx FILE *** -->
```
Comment (Peter-Metz) [inline docs/tc_overview.html]: And here

File: docs/tc_roadmap.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!-- *** NEVER EDIT THIS FILE BY HAND *** -->
-<!-- *** INSTEAD EDIT docs/tc_roadmap.htmx *** -->
+<!-- *** INSTEAD EDIT /Users/matthewjensen/Projects/Tax-Calculator/docs/tc_roadmap.htmx FILE *** -->
```
Comment (Peter-Metz) [inline docs/tc_roadmap.html]: And here

File: docs/tc_starting.html [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!-- *** NEVER EDIT THIS FILE BY HAND *** -->
-<!-- *** INSTEAD EDIT docs/tc_starting.htmx *** -->
+<!-- *** INSTEAD EDIT /Users/matthewjensen/Projects/Tax-Calculator/docs/tc_starting.htmx FILE *** -->
```
Comment (Peter-Metz) [inline docs/tc_starting.html]: And here

**Comment by Peter-Metz** [comment]
@MattHJensen I think a few file paths got changed that shouldn't have, but everything else looks good.

**Comment by MattHJensen** [comment]
Thanks @Peter-Metz. I will merge this as soon as the tests complete and then proceed with the release.

**Pull request merged**